Frameworks/CoreGraphics/CGImage.mm

```diff
@@ -490,8 +490,9 @@ - (NSUInteger)length {
 CGDataProviderRef CGImageGetDataProvider(CGImageRef img) {
     const UInt8* pPtr = (const UInt8*)img->Backing()->LockImageData();
     CFIndex length = img->Backing()->Height() * img->Backing()->BytesPerRow();
-    woc::unique_cf<CFDataRef> data{ CFDataCreateWithBytesNoCopy(nullptr, pPtr, length, kCFAllocatorNull) };
-    CGDataProviderRef dataProvider = CGDataProviderCreateWithCFData(data.get());
+
+    //TODO 1709:: Autorelease dataProvider so it won't leak for consumers expecting a non-owning reference
```

Comment:
the actual fix is postponed till the merge, is it too much work to actually fix the ImageIO test rather than introducing intentional bugs to satisfy bad test code?